app/schemas/archive-format.json

```diff
@@ -0,0 +1,222 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://the-federation.info/specs/backup-restore#archive_format",
```

Comment:
This document is likely to be discontinued and the ID should be changed. But I'm not sure how to make a new one.